distributed/client.py

```diff
@@ -1373,15 +1373,20 @@ def _dec_ref(self, key):
                 del self.refcount[key]
                 self._release_key(key)
 
-    def _release_key(self, key):
+    def _release_key(self, key, stimulus_id=None):
```

Comment:
This function should no longer need a stimulus_id kwarg, does it? The contextvar should be set

Reply:
This is on the client side, which is not using ContextVars yet.